data/json/vehicleparts/tanks.json

```diff
@@ -0,0 +1,277 @@
+[
+  {
+    "abstract": "tank_little",
+    "type": "vehicle_part",
+    "name": "vehicle tank (2L)",
+    "item": "metal_tank_little",
+    "location": "fuel_source",
+    "symbol": "O",
+    "broken_color": "red",
+    "size": 2000,
+    "durability": 75,
+    "difficulty": 1,
+    "damage_modifier": 80,
+    "breaks_into": [ { "item": "scrap", "count": [ 1, 2 ] } ],
+    "flags": [ "FUEL_TANK" ]
+  },
+  {
+    "abstract": "tank_small",
+    "type": "vehicle_part",
+    "name": "vehicle tank (10L)",
+    "item": "jerrycan",
+    "location": "fuel_source",
+    "symbol": "O",
+    "broken_color": "red",
+    "size": 10000,
```

Comment:
The changed vehicle parts still have "size" property, but the documentation in veh_type.h indicates the size is only used for cargo parts. Indeed, `vpart_info::size` is not checked for fuel tanks. If it's not needed, it should be removed as to avoid mistakes.